service/src/main/kotlin/app/cash/backfila/client/HttpClientServiceApi.kt

```diff
@@ -41,6 +41,6 @@ interface HttpClientServiceApi {
   ): RunBatchResponse
 
   companion object {
-    private const val BASE_PATH = "/backfila"
```

Comment:
Shouldn't you have some other change potentially that ensures the base path has a trailing `/`? I would like to see some tests for this as well.

Reply:
Yeah I'd like a couple tests. Particularly to prove that this won't break existing clients.

Reply:
may be hard to test since clients specify the path, but this is service code. It seems fine to me

Reply:
Yeah true. As long as we check that existing clients are fine with the missing slash that will probably have to be enough.

Maybe try using the backfila Development app with existing misk Development app